### ISSUE TITLE
Instant preview image (link share) 🤔

### ISSUE THREAD

**Issue by PositiveUser** [juxtopposed/realtimecolors#66]
I always see the same below image after sharing a link in Telegram.

![image](https://github.com/juxtopposed/realtimecolors/assets/89864580/11abbe9b-9230-49be-b565-42495a3c57d2)
<br>
I don't want this;
I want [the theme I shared on Telegram](https://www.realtimecolors.com/?colors=e0e0e0-221f1f-51c4ae-363030-7edd69) to be reflected in Telegram.
Like in below image:

![image](https://github.com/juxtopposed/realtimecolors/assets/89864580/bc010a2f-f5e4-498a-96c5-0e978d66df1d)

Plus, what I currently see is an image of the old version of Realtime Colors.

This allows for instant preview without opening the link.
However only some sites may have such a feature.
And it's okay if you don't implement this.

**Comment by FleetAdmiralJakob** [comment]
Yes, it would be cool to get a dynamic preview image @juxtopposed. I also share the colors with some of my friends and would need this feature.
